evm_arithmetization/src/cpu/kernel/interpreter.rs

```diff
@@ -664,9 +759,6 @@ impl<F: Field> Interpreter<F> {
         .len()
             > offset
         {
-            // Even though we are in the interpreter, `JumpdestBits` is not part of the
-            // preinitialized segments, so we don't need to carry out the additional checks
-            // when get the value from memory.
```

Comment:
I thought you argued against removing these comments?

Reply:
But with the switch to the `State` trait, there was no boolean regarding whether we are in the witness generation or the interpreter anymore (we moved the preinitialized segments to the `MemoryState` directly, so the checks are dealt with directly there).